### PR TITLE
add 'contact info' and 'revision' to title page keywords

### PR DESCRIPTION
**Pull request by kortina**
When I created a fountain file with Highland 2 and generated the title page, I noticed the keywords were not correctly highlighting.

I realized this was because "contact" was supported, but not "contact info." I also noticed "revision" was not supported.

title page generator from highland:

image

highland editor:

image

vscode editor:

image

vscode properly highlighting "contact":

image

### CODE REVIEW

**Comment by piersdeseilligny** [comment]
Thanks for the PR! I'il look into this a bit more tomorrow - what does it look like in the rendered PDF and live preview?

**Comment by kortina** [comment]
Hmm, I'm not getting it to show up on the print view in the right place:

image

See this diff with a few more changes: https://github.com/piersdeseilligny/betterfountain/pull/122/commits/8ea858d702d513dfd7ae79f754e77fe658f29920

I also tried updating the Completion file: https://github.com/piersdeseilligny/betterfountain/pull/122/commits/0be83cebd893d30020ef65852d1e359f86ad3c73

Any ideas on how to get the preview working for this?

File: .vscode/settings.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
 	"typescript.tsdk": "./node_modules/typescript/lib",
 	"typescript.tsc.autoDetect": "off",
 	"npm.exclude": ["**/client", "**/server"],
-	"npm.enableScriptExplorer": true
+	"npm.enableScriptExplorer": true,
+	"editor.formatOnSave": false,
```
Comment (kortina) [inline .vscode/settings.json:12]: Our auto-formatters are different, so turned this off. Lmk if you want me to remove

**Comment by piersdeseilligny** [comment]
Thanks for this!! Unfortunately my laptop's motherboard died a few days ago, so it's gonna be a few days at least before I can review this, but i'il definitely let you know!

**Comment by piersdeseilligny** [comment]
So I believe this is mostly fixed, the main issue is that you missed [this small line](https://github.com/piersdeseilligny/betterfountain/blob/master/src/afterwriting-parser.ts#L308) :D all the title page keys get any spaces replaced by an underscore, so it should have been `contact_info` rather than `contact info`. I also fixed some of the regex.

Let me know if you can see any other issues with this, or if it's good to merge?

**Comment by kortina** [comment]
Sorry, this got away from me. Your changes make sense. Let's get it merged?

**Pull request merged**